selfdrive/controls/controlsd.py

```diff
@@ -578,6 +578,13 @@ def state_control(self, CS):
     sr = max(params.steerRatio, 0.1)
     self.VM.update_params(x, sr)
 
+    # Update Torque Params
+    if self.CP.lateralTuning.which() == 'torque':
+      torque_params = self.sm['liveTorqueParameters']
+      # Todo: Figure out why this is needed, and remove it
+      if (torque_params.latAccelFactorFiltered > 0) and (self.sm.valid['liveTorqueParameters']):
```

Comment:
also check liveTorqueParameters.liveValid or is that for offline debugging?

Reply:
For offline debugging